Formula/nghttp2.rb

```diff
@@ -1,9 +1,11 @@
 class Nghttp2 < Formula
```

Comment:
```suggestion
class Nghttp2 < Libnghttp2
```

I'm wondering if it's worth trying this some time and seeing if it works to only override `install` and `test`. Not blocking this PR at all, though.

Reply:
I quite like that, though I do suspect it might require some tweaks elsewhere so that a version bump for `libnghttp2` will trigger a build for `nghttp2`.